cpp/src/io/parquet/reader_impl_preprocess.cu

```diff
@@ -271,8 +272,10 @@ void generate_depth_remappings(std::map<int, std::pair<std::vector<int>, std::ve
   DecodePageHeaders(chunks.device_ptr(), chunks.size(), error_code.data(), stream);
   chunks.device_to_host_sync(stream);
 
-  if (error_code.value() != 0) {
-    CUDF_FAIL("Parquet header parsing failed with code(s) " + error_code.str());
+  if (error_code.value() != 0 and
```

Comment:
Might want to add a comment here to explain why we're ignoring unsupported encoding errors. Maybe tag this PR in the comment too.

Reply:
Good point, will push that